contracts/src/v0.8/dev/automation/2_1/KeeperRegistryLogicB2_1.sol

```diff
@@ -284,6 +285,11 @@ contract KeeperRegistryLogicB2_1 is KeeperRegistryBase2_1 {
     return abi.decode(s_upkeepTriggerConfig[upkeepId], (LogTriggerConfig));
   }
 
+  function getBlockTriggerConfig(uint256 upkeepId) public view returns (BlockTriggerConfig memory) {
+    require(getTriggerType(upkeepId) == Trigger.LOG);
+    return abi.decode(s_upkeepTriggerConfig[upkeepId], (BlockTriggerConfig));
+  }
+
```

Comment:
I thought we were going to remove this?

Reply:
leaving it for now because nodes are already using it and I don't want to unnecessarily introduce breaking changes. We can remove in a future cleanup push or slap a deprecation warning on it for now. (I'd prefer the latter).

Reply:
I'm not sure i agree, we are restricting ourselves because of previous decisions and having contract bloat

Reply:
agreed that keeping for now, will remove in future PR